src/theme.ts

```diff
@@ -121,4 +125,6 @@ export default createMuiTheme({
   sidebarMobilePadding: 8,
   participantBorderWidth: 2,
   mobileTopBarHeight: 52,
+  chatWindowWidth: 320,
+  chatWindowHeaderHeight: 56,
```

Comment:
I think it would be ok if this value is stored inside the `ChatWindowHeader` component itself. I think the theme object is a good place to put values that will be used in multiple spots. I think that the `chatWindowHeaderHeight` value is only going to be used by that component.

I think it's fine to keep `chatWindowWidth` in here though.

Reply:
i will keep that in mind! removed and updated ChatWindowHeader